test/adapters/file/write.spec.js

```diff
@@ -111,4 +112,32 @@ describe('write file adapter tests', function () {
         });
     });
 
+    _.each(['csv', 'jsonl'], function(format) {
+        it('can write a point and read it back with ' + format + ' format', function() {
+            return check_juttle({
+                program: 'emit -limit 1 ' +
```

Comment:
Multi-line template strings are the future! See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings.